src/coreclr/jit/lower.cpp

```diff
@@ -9046,7 +9050,7 @@ const int LDP_REORDERING_MAX_DISTANCE = 16;
 // Returns:
 //    True if the optimization was successful.
 //
-bool Lowering::OptimizeForLdp(GenTreeIndir* ind)
+bool Lowering::OptimizeForLdpStp(GenTreeIndir* ind)
```

Comment:
Can we move this method in `lowerarmarch.cpp`?

Reply:
Also, please update the method docs.

Reply:
Let me do that separately, otherwise the diff won't be readable.

Reply:
> Also, please update the method docs.

Do you mind if I include that in the move follow-up as well?

Reply:
sure

Reply:
I'm not positive moving it into `lowerarmarch` is goodness.

We're likely going to move the logic back out for RV64/LA64 when they eventually get that far and for APX on xarch (new instructions exist such as push2/pop2)